src/main/java/net/snowflake/ingest/streaming/internal/ColumnMetadata.java

```diff
@@ -128,6 +129,15 @@ public Integer getOrdinal() {
     return ordinal;
   }
 
+  @JsonProperty("source_iceberg_data_type")
```

Comment:
mind adding a comment here with a sample content of what goes into this field?

Reply:
Added at variable declaration.